src/ResourceManager/OperationalInsights/Commands.OperationalInsights/packages.config

```diff
@@ -1,4 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.Azure.Management.OperationalInsights" version="0.18.0-preview" targetFramework="net45" />
-</packages>
+  <package id="Microsoft.Azure.OperationalInsights" version="1.0.0-preview" targetFramework="net452" />
+  <package id="Microsoft.CSharp" version="4.3.0" targetFramework="net452" />
```

Comment:
Remove the following four packages, they will be imported with common dependencies.

Reply:
Fixed.